tests/suites/static_analysis/lint_go.sh

```diff
@@ -38,6 +38,20 @@ run_domain_imports() {
 	done
 }
 
+run_juju_errors_imports() {
+    pkgs=("domain")
+
+    for pkg in "${pkgs[@]}"; do
+        dirs=$(find ${pkg} -mindepth 1 -maxdepth 10 -type d | sort -u)
+        for subdir in $dirs; do
+            echo "$subdir"
+            imports=$(go list -json -e -test "./${subdir}" 2>/dev/null | jq -r ".Imports // [] | .[]")
+            disallowed="github.com/juju/errors"
+            python3 tests/suites/static_analysis/lint_go.py -d "${disallowed}" -g "${imports}" || (echo "Error: pkg $subdir contains juju/errors imports" && exit 1)
```

Comment:
Like this error message better than the ones used in the other import checks. Any chance of a drive by improvement?

Reply:
Done & Done